index.html

```diff
@@ -1183,7 +1183,7 @@
 should restrict the [=data=] they transfer to what's either necessary to achieve their users'
 goals or aligns with their users' wishes and interests.</span></div>
 
-<div class="practice" data-audiences="api-designers"><span class="practicelab">Web APIs should be designed to minimize the amount of data that sites need
+<div class="practice" data-audiences="api-designers"><span class="practicelab">Web APIs must be designed to minimize the amount of data that sites need
```

Comment:
This one has tradeoffs: letting callers minimize the data increases the size of the API ... so I think it should remain "should".